src/main/cpp/Robot.cpp

```diff
@@ -2,13 +2,25 @@
 
 #include "Robot.h"
 #include "RobotContainer.h"
+#include "frc/TimesliceRobot.h"
 
 #include <frc2/command/CommandScheduler.h>
 #include <frc/DataLogManager.h>
 #include <frc/DriverStation.h>
 
 using namespace frc;
 
+Robot::Robot() : frc::TimesliceRobot{2_ms, 5_ms} {
+  // Schedule periodic functions
+  Schedule(
+    [=] {
```

Comment:
In C++20, implicit capture of this is deprecated:
```suggestion
    [=, this] {
```